frontend/src/pages/Seller/Auth/ForgotPassword.tsx

```diff
@@ -1,9 +1,70 @@
-import * as React from "react"
-const ForgotPassword: React.FC<{}> = (props) => {
+
+
+import * as React from "react";
+import { Input, Button, Box, Flex } from '@chakra-ui/react';
+import './Login.css'; // Import the shared CSS file
+
+interface ForgotPasswordProps {}
+
+const ForgotPassword: React.FC<ForgotPasswordProps> = () => {
+    const [newPassword, setNewPassword] = React.useState<string>('');
+    const [confirmNewPassword, setConfirmNewPassword] = React.useState<string>('');
+
+    const handleResetPassword = () => {
+
+        console.log('Resetting password with:', { newPassword, confirmNewPassword });
+    };
+
     return (
-        <div>
-            <h1>Forgot password</h1>
-        </div>
+        <Flex className="content">
+
+            <Box className="gradient-bg" />
+
+
+            <Box className="login-container">
+                <h1 className="login-header">Forgot Password</h1>
+
+
+                <Input
+                    placeholder="New Password"
+                    type="password"
+                    value={newPassword}
+                    onChange={(e) => setNewPassword(e.target.value)}
+                    mb={4}
+                />
+
+
+                <Input
+                    placeholder="Confirm New Password"
+                    type="password"
+                    value={confirmNewPassword}
+                    onChange={(e) => setConfirmNewPassword(e.target.value)}
+                    mb={6}
+                />
+
+
+                <Button
```

Comment:
Add icons to the buttons as per the designs. I added a resource for this in the last PR review.

Reply:
Use icons similar to those present in the designs, but they don't have to be exactly the same.